apps/console/src/app.tsx

```diff
@@ -43,6 +44,7 @@ import { initializeAuthentication } from "./features/authentication";
 import { EventPublisher, PreLoader } from "./features/core";
 import { ProtectedRoute } from "./features/core/components";
 import { Config, getBaseRoutes } from "./features/core/configs";
+import documentationLinks from "./features/core/configs/documentation";
```

Comment:
Lets make `documentationLinks` a named export.
```suggestion
import documentationLinks from "./features/core/configs";
```

Reply:
Fixed with 8be4530f6408250e5393750b9dbd6a422bc4367f